### PR TITLE
Publish Modal UI update

### PR DESCRIPTION
**Pull request by guptaharsh25**
[AB#34241](https://microsoftgarage.visualstudio.com/002806e2-ebaa-4672-9d2e-5fe5d29154ef/_workitems/edit/34241)

NOTE: Please review this PR first https://github.com/microsoft/adaptivecards-templates/pull/223

Updated Publish Modal UI to match designs and localized strings in modal.

image

### CODE REVIEW

File: private-templates-service/client/src/components/Common/PublishModal/styled.tsx [hunk 1]
```diff
@@ -25,23 +25,27 @@ export const Modal = styled.div`
 `;
 
 export const Header = styled.div`
-  font-size: 1.375rem;
+  font-family: ${FONTS.SEGOE_UI_SEMI_BOLD};
```
Comment (daniellemott) [inline private-templates-service/client/src/components/Common/PublishModal/styled.tsx:28]: Thanks :)

File: private-templates-service/client/src/components/Common/PublishModal/styled.tsx [hunk 2]
```diff
@@ -25,23 +25,27 @@ export const Modal = styled.div`
...
+  font-size: 1.75rem;
   font-weight: 500;
   margin-bottom: 12px;
 `;
 
 export const Description = styled.div`
+  font-family: ${FONTS.SEGOE_UI_REGULAR};
```
Comment (daniellemott) [inline private-templates-service/client/src/components/Common/PublishModal/styled.tsx:35]: Nit: Do we need to explicitly specify the SEGOE_UI_REGULAR font? I think it's just inherited from the body?

Comment (guptaharsh25) [reply private-templates-service/client/src/components/Common/PublishModal/styled.tsx:35]: Hmm wasn't sure, I can remove it if it's unnecessary

Comment (daniellemott) [reply private-templates-service/client/src/components/Common/PublishModal/styled.tsx:35]: Does the font look different if you remove it? If not, I would say remove it

**Pull request merged**